applications/GeoMechanicsApplication/custom_elements/geo_structural_base_element.cpp

```diff
@@ -601,6 +467,23 @@ SizeType GeoStructuralBaseElement<TDim, TNumNodes>::GetAlongNumberIntegrationPoi
     return 0;
 }
 
+template <unsigned int TDim, unsigned int TNumNodes>
+Element::DofsVectorType GeoStructuralBaseElement<TDim, TNumNodes>::GetDofs() const
+{
+    auto result = Element::DofsVectorType{};
+    for (const auto& r_node : GetGeometry()) {
+        result.push_back(r_node.pGetDof(DISPLACEMENT_X));
+        result.push_back(r_node.pGetDof(DISPLACEMENT_Y));
+        if constexpr (TDim == 3) {
+            result.push_back(r_node.pGetDof(DISPLACEMENT_Z));
+            result.push_back(r_node.pGetDof(ROTATION_X));
+            result.push_back(r_node.pGetDof(ROTATION_Y));
+        }
+        result.push_back(r_node.pGetDof(ROTATION_Z));
```

Comment:
Which makes clear that the structural base element is a beam element.

Reply:
Agreed. This is something to keep an eye on when we move over to the structural elements of the `StructuralMechanicsApplication` (since the "geo" structural elements will be removed at some point in time).